store.html

```diff
@@ -0,0 +1,176 @@
+              <p class="lastname" for="lname">Last name*</p>
+              <input type="text" id="lname" name="lname" />
+            </div>
+          </section>
+          </article>
+            <!----names-box--end----->
+
+
+          <!----address-box----->
+          <article class="address-box" >
+
+            <div class="inputfild">
+              <p for="address">Address</p>
+              <input type="text" id="add-box" name="address" />
+            </div>
+            
+            <div class="inputfild">
+              <p for="address">Address*</p>
+              <input type="text" id="add-box" name="address" />
+            </div>
+          </article>
+           <!----address-box--end----->
+
+
+
+              <!----citypost------>
+            <article class="citypost">
+              <section class="city">
+              <div class="inputfild">
+                <p for="city">City*</p>
+                <input type="text" id="lname" name="fname" placeholder="Select your city..."/>
+              </div>
+            </section>
+  
+              
+              <section class="post">
+              <div class="inputfild">
+                <p class="lastname" for="Postcode">Postcode*</p>
+                <input type="text" id="lname" name="lname" />
+              </div>
+            </section>
+            </article>
+              <!----citypost-end----->
+
+
+
+
+                 <!----select-color------>
+                 <article class="colors">
+                 <article class="color">
+                    <p><strong>Select a colour</strong></p>
+                 </article>
+                 
+                  <p>
+                 <input id="karma-orange" type="radio" name="size" value="Karma orange" checked />
+                 <label for="karma-orange">Karma orange</label>
+                 
+
+                 <input id="space-grey" type="radio" name="size" value="Space Grey" />
+                 <label for="Space Grey">Space Grey</label><br>
+                 
+                 <section class="agree">
+                 <input id="agreement" type="checkbox" name="agreement" checked />
+                 <label  for="agreement"><span class="span1">By placing your order you agree to Karma's </span><ins><span style="color:#DE6E49">Terms and <br><u style="color:#DE6E49">Conditions.</u>*</ins></label>
+                 
+                </section>   
+                </article>
+                <!----select-color-end----->
+
+
+                  <!-------BUTTON-------->
+
+            <section class="main">
+      
+              <div class="btn">
+               <button class="button" target="_blank" >Place my order</button>
+              </div>
+            </section>
+                   <!-------BUTTON-END------->
+
+                 
+        </div>  <!-----labels----->
+      </section> <!--end of order-->
+      
+    </section> <!---end of row---
+
+
+
+    <!-------footer-------->
```

Comment:
overall I think you did a great job , really like this commenting out different sections to make it much easier to read,Well done!